apps/derive/elpi/derive.elpi

```diff
@@ -4,82 +4,118 @@
 
 namespace derive {
 
+pred exists-indc i:inductive, i:(constructor -> prop).
+exists-indc I P :-
+  coq.env.indt I _ _ _ _ KL _,
+  std.exists KL P.
+
 pred if-verbose i:prop.
-if-verbose P :- get-option "verbose" tt, !, P.
+if-verbose P :- (get-option "verbose" tt ; get-option "recursive" tt), !, P.
```

Comment:
Is it done on purpose, or was it introduced for debugging reasons?

Reply:
I did it on purpose, since I believe `recursive` is a sort of debugging option.